src/modules/ProductLicense.rb

```diff
@@ -1144,12 +1144,24 @@ def AskLicenseAgreement(src_id, dir, patterns, action, enable_back, base_product
       end
 
       licenses_ref = arg_ref(licenses)
-      DisplayLicenseDialog(
+
+      label = Pkg::SourceGeneralData(src_id)["name"]
+      title = _("License Agreement")
+
+      if !label.empty?
```

Comment:
Add a comment for translators, describe what the `%s` mean so they can translate it properly.

    # TRANSLATORS: %s is a repository name